### ISSUE TITLE
Where are the Mail?

### ISSUE THREAD

**Issue by ohobby** [mailcow/mailcow-dockerized#1057]
Hi,
I just noticed that probably some mails do not arrive, although they would have to.
A mail from flickr is listed in rspamd with no action, but not in show in SoGo or imap mail.
Testmail produced via flickr, which landed in the spam folder because of score over 5.
This moved to the inbox and thus learned as HAM. The next testmail from flickr will appear in rspamd again with no action, but will not arrive in the Inbox.
I start to find out where the Mail are lost, but i have not found a point. :-(

Any ideas or tips on where to look?

Thanks,
Oliver

**Comment by ohobby** [comment]
If i whitelist *@flickr.com the Testmail also do not reach the inbox.

**Comment by andryyy** [comment]
Do you use tagged mail adresses like me+flickr@domain?

Check or paste the Dovecot and Postfix logs, please.

**Comment by ohobby** [comment]
In that case i don't use tagged mail a.dress.
Today evening i make some additional test and post then the logs, because i cant see reasons in the logs.
The strange thing is, that if i insert a long text in the body, the mail reach the inbox.

**Comment by ohobby** [comment]
Hier ein Auszug aus den Logs von einer Mail, die nicht in der Inbox auftaucht. (*@flickr.com ist zur Zeit whitelistet)

POSTFIX:

postfix-mailcow_1    | Feb 16 18:59:32 mail postfix/postscreen[3940]: CONNECT from [74.6.128.145]:39893 to [172.22.1.3]:25
postfix-mailcow_1    | Feb 16 18:59:32 mail whitelist_forwardinghosts: Look up 74.6.128.145 on whitelist, result 200 DUNNO
postfix-mailcow_1    | Feb 16 18:59:32 mail postfix/dnsblog[3954]: addr 74.6.128.145 listed by domain hostkarma.junkemailfilter.com as 127.0.0.3
postfix-mailcow_1    | Feb 16 18:59:32 mail postfix/dnsblog[3954]: addr 74.6.128.145 listed by domain hostkarma.junkemailfilter.com as 127.0.1.1
postfix-mailcow_1    | Feb 16 18:59:34 mail postfix/postscreen[3940]: PASS OLD [74.6.128.145]:39893
postfix-mailcow_1    | Feb 16 18:59:34 mail postfix/smtpd[3956]: connect from sonic328-19.consmr.mail.bf2.yahoo.com[74.6.128.145]
postfix-mailcow_1    | Feb 16 18:59:35 mail postfix/smtpd[3956]: Anonymous TLS connection established from sonic328-19.consmr.mail.bf2.yahoo.com[74.6.128.145]: TLSv1.2 with cipher ECDHE-RSA-AES128-GCM-SHA256 (128/128 bits)
postfix-mailcow_1    | Feb 16 18:59:35 mail postfix/smtpd[3956]: 8CCC7A059A: client=sonic328-19.consmr.mail.bf2.yahoo.com[74.6.128.145]
postfix-mailcow_1    | Feb 16 18:59:35 mail postfix/cleanup[3960]: 8CCC7A059A: message-id=<>
postfix-mailcow_1    | Feb 16 18:59:36 mail postfix/qmgr[339]: 8CCC7A059A: from=<flickr-wl.mzwgsy3loiwxo3bnge2tcobygaztsnzq-@returns.bulk.yahoo.com>, size=28745, nrcpt=1 (queue active)
postfix-mailcow_1    | Feb 16 18:59:36 mail postfix/lmtp[3961]: 8CCC7A059A: to=<oliverr@ohobby.de>, relay=dovecot[172.22.1.4]:24, delay=0.96, delays=0.89/0.01/0.01/0.05, dsn=2.0.0, status=sent (250 2.0.0 <oliverr@ohobby.de> SNN4DAgch1pYbQAAaMj/fg Saved)
postfix-mailcow_1    | Feb 16 18:59:36 mail postfix/qmgr[339]: 8CCC7A059A: removed
postfix-mailcow_1    | Feb 16 18:59:36 mail postfix/smtpd[3956]: disconnect from sonic328-19.consmr.mail.bf2.yahoo.com[74.6.128.145] ehlo=2 starttls=1 mail=1 rcpt=1 data=1 quit=1 commands=7
postfix-mailcow_1    | Feb 16 19:02:56 mail postfix/anvil[3958]: statistics: max connection rate 1/60s for (smtpd:74.6.128.145) at Feb 16 18:59:34
postfix-mailcow_1    | Feb 16 19:02:56 mail postfix/anvil[3958]: statistics: max connection count 1 for (smtpd:74.6.128.145) at Feb 16 18:59:34

DOVECOT:
dovecot-mailcow_1    | Feb 16 18:59:36 mail dovecot: lmtp(27992): Connect from 172.22.1.3
dovecot-mailcow_1    | Feb 16 18:59:36 mail dovecot: lmtp(oliverr@ohobby.de): SNN4DAgch1pYbQAAaMj/fg: sieve: msgid=unspecified: stored mail into mailbox 'INBOX'
dovecot-mailcow_1    | Feb 16 18:59:36 mail dovecot: lmtp(27992): Disconnect from 172.22.1.3: Successful quit

**Comment by ohobby** [comment]
Hallo @andryyy ,
für mich sehen die Logfiles gut aus und deswegen ist die Frage, wo ist/sind die Mails?
Habe nichts dagegen, das Mails irgendwo gefiltert werden, aber dann sollte man es in den Logfiles sehen.
Grüße

**Comment by andryyy** [comment]
We do not delete or drop any mail that is not rejected... please check your setup. Your mail is stored to INBOX (see logs), you should check /var/lib/docker/volumes/mailcow...vmail/_date/var/vmail/domain/mailbox for your mail msg. I highly doubt that a msg just vanishes.

**Comment by ohobby** [comment]
I use the default settings.
And everything looks good. :-)
And you are right the system do not delete or drop something.
Flickr was the first thing i have change to the new Mailbox and i have seen, that the most mails are not shown in SOGo or imap Mail Programm.
Now i have make additional tests and I have found the mistake. :-(
Long long Time ago i have create a Filter in my Mailgrogram, to move flickr Mail and that was the cause for the "disappeared" mails. 

Shame on me, that I did not get there earlier.

I wish you a nice weekend.
